front/hooks/useNovuClient.ts

```diff
@@ -0,0 +1,36 @@
+import { Novu } from "@novu/js";
+import { useEffect, useState } from "react";
+
+import { useUser } from "@app/lib/swr/user";
+
+export const useNovuClient = () => {
```

Comment:
That'll add a lot of boilerplate code to handle the nullability of the novuClient. 

I don't know react well can't we do something simpler?